standard-interfaces/cloud.astarte.genericsensors.Values.json

```diff
@@ -0,0 +1,17 @@
+{
+    "interface_name": "cloud.astarte.genericsensors.Values",
+    "version_major": 0,
+    "version_minor": 1,
+    "type": "datastream",
+    "ownership": "device",
+    "description": "Generic sensors sampled data.",
+    "doc": "This interface reports sensors sampled data datastreams. Each datastream is idenitified using a sensor_id.",
```

Comment:
> This interface reports sensors sampled data datastreams

wtf.

Something like "Values allows generic sensors to stream samples. It is usually used in combination with AvailableSensors, which makes the device aware of what sensors and what unit of measure they are reporting. sensor_id represents an unique identifier for an individual sensor, and should match sensor_id in AvailableSensors when used in combination."

Reply:
Ok, however I've changed `which makes the device aware ` to `which makes API client aware `.